### PR TITLE
[SCB-1633] update README

### PR DESCRIPTION
**Pull request by kakulisen**

### CODE REVIEW

File: README-ZH.md [hunk 1]
```diff
@@ -22,27 +22,27 @@ Apache ServiceComb Toolkit 是基于契约的微服务开发工具套件
 
 * 契约与代码一致性校验
 
-  校验应用的实际实现（如数据和服务API）是否与样本服务契约描述一致。
+  校验应用的实际实现（如数据和服务API）是否与样本服务契约描述一致。支持风格校验和兼容性校验,详情请见[OAS Validator](./oas-validator/README-ZH.md)
```
Comment (chanjarster) [inline README-ZH.md]: I think the style check and compatibility check feature should be listed individually

File: README-ZH.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ $ git clone https://github.com/apache/servicecomb-toolkit.git
 $ cd toolkit
 
 # 构建打包
-$ mvn clean install
+$ mvn clean install -DskipTests
```
[the remark this answers is not in the document]
Comment (kakulisen) [reply README-ZH.md]: Ok, I will fix it in next commit

File: README-ZH.md (hunk 1 above)
Comment (kakulisen) [reply README-ZH.md]: I am lazy here. Let me take a moment to think about how to organize it

**Comment by kakulisen** [comment]
@chanjarster  I have modified code according to the review suggests. please help to check it again. thanks

**Comment by chanjarster** [comment]
LGTM

**Pull request merged**